advanced/Scripts/whitelist.sh

```diff
@@ -7,83 +7,174 @@
 # the Free Software Foundation, either version 2 of the License, or
 # (at your option) any later version.
 
+if [[ $# = 0 ]]; then
+    echo "Immediately whitelists one or more domains in the hosts file"
+    echo " "
+    echo "Usage: whitelist.sh domain1 [domain2 ...]"
+    echo "  "
```

Comment:
Huh, thought i'd sorted that.. I think I might have got a bit confused with the merging earlier! Will sort it out in a bit